R/valueBoxModule.R

```diff
@@ -79,7 +79,7 @@ valueBoxModuleUI <- function(
 #'
 #' @export
 #'
-valueBoxModule <- function(input, output, session, value) {
+valueBoxModule <- function(input, output, session = shiny::getDefaultReactiveDomain(), value) {
```

Comment:
I don't think we should set this default value because we never actually call this function with this argument.  This is handled under the hood by callModule.